PoshRSJob/Public/Start-RSJob.ps1

```diff
@@ -39,7 +39,10 @@ Function Start-RSJob {
         .PARAMETER PSSnapinsToImport
             A collection of PSSnapins that will be imported into the background runspace job.
 
-        .PARAMETER FunctionsToImport
+        .PARAMETER FunctionFilesToLoad
+            A collection of files containing custom functions that will be imported into the background runspace job.
+
+        .PARAMETER FunctionsToLoad
```

Comment:
This will be a breaking change for those who have been using this parameter. I would recommend adding an alias on **FunctionsToImport** called **FunctionsToLoad**.